src/core/directives/google-map.ts

```diff
@@ -268,6 +285,17 @@ export class SebmGoogleMap implements OnChanges, OnInit {
     }
   }
 
+  private _fitBounds() {
+    if (this.fitBounds != null && !this.usePanning) {
```

Comment:
@SebastianM This seems a bit odd, you could set fitBounds to `false` and this would get triggered eh? Maybe just a simpler truthiness check will do?